bridges/otelzap/README.md

```diff
@@ -0,0 +1,11 @@
+# OpenTelemetry Zap Log Bridge
+
+[![Go Reference](https://pkg.go.dev/badge/go.opentelemetry.io/contrib/bridges/otelzap.svg)](https://pkg.go.dev/go.opentelemetry.io/contrib/bridges/otelzap)
+
+This package implements a [Log Bridge](https://opentelemetry.io/docs/specs/otel/logs/bridge-api/) for the [Zap](https://github.com/uber-go/zap) logger. This bridge can be used to send logs emitted by application using Zap to an OpenTelemetry compatible endpoint through the [OpenTelemetry Log API](https://pkg.go.dev/go.opentelemetry.io/otel/log).
+
+## Example
+
+Please refer to the [example_test.go](./example_test.go) file for a complete example of how to create a Zap logger using the bridge.
+
+
```

Comment:
```suggestion
```

package docs contain a reference to the example. I think this is not needed.

Reply:
fixed